src/api/handlers/guild.ts

```diff
@@ -558,7 +565,7 @@ export async function pruneMembers(guildID: string, options: PruneOptions) {
 
   const result = await RequestManager.post(
     endpoints.GUILD_PRUNE(guildID),
-    { ...options, include_roles: options.roles.join(",") },
+    { ...options, compute_prune_count: computePruneCount, include_roles: roles },
```

Comment:
Maybe add this comment?

```suggestion
    // TODO: v11 use camel to snake function
    { ...options, compute_prune_count: computePruneCount, include_roles: roles },
```

Reply:
Those functions do not really exist...